src/buttons/buttons.js

```diff
@@ -1,12 +1,12 @@
 angular.module('mm.foundation.buttons', [])
 
 .constant('buttonConfig', {
-    activeClass: 'active',
+    deactiveClass: 'hollow',
```

Comment:
Well, its not really deactivated. The click event is still fired. Pressing it just doesn't change the state. The hollow class indicates that the button has been pressed. So I'd say that its active or has been activated. Makes sense? I'd prefer the original activeClass.